### ISSUE TITLE
When creating folder from explorer, the newly created folder should be in selected state immediately after creation. 

### ISSUE THREAD

**Issue by ravshansbox** [Microsoft/vscode#40166]

**Comment by isidorn** [comment]
I do not agree. Since the selection in the explorer usualy shows what is opened in the editor.
Since the folder can not be opened in the editor it makes no sense to change the selection

**Comment by ravshansbox** [comment]
After creating folder, usually next step is to create files/folders inside and one have to always select folder manually after creating with the current behaviour.

**Comment by ravshansbox** [comment]
In fact, it's Atom's default behavior.

**Comment by isidorn** [comment]
Ok, that is a fair feature request -> reopening
